pycloudlib/azure/util.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of pycloudlib. See LICENSE file for license information.
+"""Azure Util Functions."""
+import logging
+
+from azure.common.client_factory import (get_client_from_cli_profile,
+                                         get_client_from_json_dict)
+from knack.util import CLIError
```

Comment:
I think we now need to add a pinned version of knack to our requirements.txt right?